### DISCUSSION TITLE
What's our workflow to support a newly released ui5 webcomponent?

### DISCUSSION THREAD

**Discussion by yzy415** [UI5/webcomponents-react#5572] [Q&A] [answered]
What's our workflow to support a newly released ui5 webcomponent?  

The ui5-webcomponent team may release a public tokenizer component described in the link below in the coming months.
 
This component has blocked a feature of my team(from SAP SF) for almost a year and we hope to deliver this feature ASAP.
 
Will it help to trigger workflow early if the consumers create a feature request or internal ticket?

https://github.com/SAP/ui5-webcomponents/issues/7003#issuecomment-1891643919

Thanks,
Alex YAn

**Comment by MarcusNotheis** [comment, marked as answer]
Hi @yzy415,

the release process for UI5 Web Components for React is more or less automated.
As soon as the UI5 Web Components are releasing a new version with a new component, we can rerun our script and release the new component as well. Usually we release our new version on the same day or the day after of the UI5 Web Components release.

--> There is no need open an issue on our side, as soon as the new web components version is available we'll publish our release as well.

Best regards,
Marcus
